sst/nights/facilities_temperatures_reports.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #89.    ax2.scatter(df_mtmount.index, df_mtmount["summaryState"], marker='|')
</pre><p>The marker could be an 'x' for better visibility.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I tried marker 'x'. However, they overlap a lot. Hence, the choice of '|'. However, I believe this part of the plot will go away.